units/crons.go

```diff
@@ -1062,6 +1062,19 @@ func PopulateSpawnhostExpirationCheckJob() amboy.QueueOperation {
 	}
 }
 
+// PopulateCloudCleanupJob returns a QueueOperation to enqueue a CloudCleanup job for Fleet in the default EC2 region.
+func PopulateCloudCleanupJob(env evergreen.Environment) amboy.QueueOperation {
+	return func(ctx context.Context, queue amboy.Queue) error {
+		return errors.Wrap(
+			amboy.EnqueueUniqueJob(
+				ctx,
+				queue,
+				NewCloudCleanupJob(env, utility.RoundPartOfHour(0).Format(TSFormat), evergreen.ProviderNameEc2Fleet, evergreen.DefaultEC2Region)),
```

Comment:
Originally I had it iterating over the ec2 regions, but then I thought that we only start Fleet hosts in the default region and that if so there isn't a way for there to be a launch template anywhere else. It seemed similar to how it's only running Cleanup for Fleet since it noops for the rest of the providers.